source/NVDAObjects/UIA/__init__.py

```diff
@@ -1186,13 +1186,21 @@ def findOverlayClasses(self,clsList):
 		if self.windowClassName == "ConsoleWindowClass":
 			from . import winConsoleUIA
 			winConsoleUIA.findExtraOverlayClasses(self, clsList)
-		elif UIAClassName in ("TermControl", "TermControl2"):
-			# TermControl2 was going to represent a terminal that supported UIA
-			# notifications (i.e. one where microsoft/terminal#12358 has been
-			# merged). However, the UIA class name was not changed in
-			# microsoft/terminal#12358 due to backward compat concerns raised
-			# by Freedom Scientific. However, a check for it is kept here just
-			# in case it should later become necessary to change it.
+
+		#: TermControl represents an up-to-date version of the UWP (standard)
+		# Windows Terminal control. TermControl2 was going to represent a
+		# terminal that supported UIA notifications (i.e. one where
+		# microsoft/terminal#12358 has been merged). However, the UIA class
+		# name was not changed in microsoft/terminal#12358 due to backward
+		# compat concerns raised by Freedom Scientific. However, a check for
+		# it is kept here just in case it should later become necessary to
+		# change it.
+		# WPFTermControl represents an embedded Windows Terminal control
+		# In .NET apps, such as LTS Visual Studio.
+		# WPFTermControl does not follow the same update cadence as TermControl
+		# and is anticipated to be replaced by the UWP implementation.
+		_all_wt_UIAClassNames = frozenset("TermControl", "TermControl2", "WPFTermControl")
+		elif UIAClassName in _all_wt_UIAClassNames:

```

Comment:
I was thinking something more like this, if we expect this list to grow, it will be easier to diff items like this.

```suggestion
		_all_wt_UIAClassNames = frozenset(
			# TermControl represents an up-to-date version of the UWP (standard)
			# Windows Terminal control.
			"TermControl",
			# TermControl2 was going to represent a
			# terminal that supported UIA notifications (i.e. one where
			# microsoft/terminal#12358 has been merged). However, the UIA class
			# name was not changed in microsoft/terminal#12358 due to backward
			# compat concerns raised by Freedom Scientific. However, a check for
			# it is kept here just in case it should later become necessary to
			# change it.
			"TermControl2",
			# WPFTermControl represents an embedded Windows Terminal control
			# In .NET apps, such as LTS Visual Studio.
			# WPFTermControl does not follow the same update cadence as TermControl
			# and is anticipated to be replaced by the UWP implementation.
			"WPFTermControl",
		)
		elif UIAClassName in _all_wt_UIAClassNames:
```